survey-results/Readme.md

```diff
@@ -0,0 +1,318 @@
+
+### Existing Gaps
+
+Next we asked what are the current gaps in the ecosystem when working on CLI applications, and what the respondent did about them. The results can be summed up into these bullets (not necessarily in order):
+
+* Better guides/examples for CLI application structure
+* (Cross Platform) Configuration File Management
+  * Better ergonomics of merging various levels of configs (global, project, CLI flags)
+* Better ergonomics of File Paths
+* Release/Packaging guides/examples
+  * More Docs/Tooling around Binary Releases
+* Clean readline implementation, problems with each current offering
+* Standardize generation / installation of manpages
+* Better CLI testing
+* Terminal Coloring (esp. Windows)
+* Better logging (or decrease current complexities)
+* Better documentation of crates
+* More libs to 1.0
+* Improve Error Handling
+* Discover ability of crates
+
+The raw word cloud looks like this (link contains higher resolution image): 
+
+[![Bubble chart showing gaps in the ecosystem](gaps.svg)](gaps.svg)
+
+Note that word cloud contains a mixture of existing gaps, what respondents did, etc. But it still paints an accurate picture of what's happening in the Rust ecosystem *today*.
+
+To begin to address some of these areas we've opened the following issues:
+
+* [Config file management · Issue #7 · rust-lang-nursery/cli-wg · GitHub](https://github.com/rust-lang-nursery/cli-wg/issues/7)
+* [Testing CLI apps · Issue #9 · rust-lang-nursery/cli-wg · GitHub](https://github.com/rust-lang-nursery/cli-wg/issues/9)
+* [Cross-platform file system abstractions · Issue #10 · rust-lang-nursery/cli-wg · GitHub](https://github.com/rust-lang-nursery/cli-wg/issues/10)
+* [Embrace Standardized Color Control · Issue #15 · rust-lang-nursery/cli-wg · GitHub](https://github.com/rust-lang-nursery/cli-wg/issues/15)
+* [Documenting CLIs · Issue #23 · rust-lang-nursery/cli-wg · GitHub](https://github.com/rust-lang-nursery/cli-wg/issues/23)
+* [Error handling in CLI apps · Issue #12 · rust-lang-nursery/cli-wg · GitHub](https://github.com/rust-lang-nursery/cli-wg/issues/12)
+* [Improve ecosystem discoverability · Issue #11 · rust-lang-nursery/cli-wg · GitHub](https://github.com/rust-lang-nursery/cli-wg/issues/11)
+
+It also appeared to be a common theme that people simply wanted to know, "What crate should I [at least start with] to do `X`?" As at times there are many crates with similar goals at appear to be competing. While I don't believe the Rust team wants to "bless" particular crates as that makes it arbitrarily hard for newcomers that are actually better to "dethrone" any currently widely used crates, we *can* do a better job at making it clearly visible which crates are actively maintained and widely used. This is somewhat orthogonal to the CLI-WG's goals, as it crosses over into recent work being done by the crates.io team, but it's still something we want to consider and work towards at all ends. 
```

Comment:
"As at times there are many crates with similar goals at appear to be competing." -> "As at times there are many crates with similar goals **th**at appear to be competing."